source/scripts/data/monsters.js

```diff
@@ -12,5 +12,23 @@ export default {
         color: DATA.COLORS.BLUE,
         health: 2,
         strength: 1,
-    }
+    },
+    BLUE_ORC: {
+        sprite: DATA.SPRITES.MONSTERS.ORC,
+        color: DATA.COLORS.BLUE,
+        health: 2,
+        strength: 1,
+    },
+    RED_ORC: {
+        sprite: DATA.SPRITES.MONSTERS.ORC,
+        color: DATA.COLORS.RED,
+        health: 1,
+        strength: 1,
+    },
```

Comment:
It's a stupid complaint, but could I get these in order from weakest to strongest?